tests/test_network.py

```diff
@@ -39,6 +43,19 @@ def test_add_data_sample(factory, session):
     session.add_data_sample(spec)
 
 
+def test_add_data_sample_located_in_shared_path(factory, session, node_cfg):
+    spec = factory.create_dataset()
+    dataset = session.add_dataset(spec)
+
+    spec = factory.create_data_sample(test_only=True, datasets=[dataset])
+
+    # move data sample to substra backend shared file system
+    with tempfile.TemporaryDirectory(prefix=node_cfg.shared_path + '/') as dst:
```

Comment:
We should not delete data sample path.
When adding assets in other tests their data are not deleted, it should be the same here.

If needed deleting the node "data" files could be done at the end of each tests, all tests? But this should be done in a different PR.